demo/perf/src/main/resources/microservice.yaml

```diff
@@ -15,28 +15,47 @@
 ## limitations under the License.
```

Comment:
META-INF  LICENCE.txt & NOTICE.txt files are missing

Reply:
done

Reply:
Hi, you don't need add the LICENCE.txt & NOTICE.txt by hand any more.  There is a maven plugin which could take care of these license files when building the artifacts.

Reply:
deleted